classifier/bisac.py

```diff
@@ -0,0 +1,601 @@
+        m(Personal_Finance_Investing, nonfiction, 'Business & Economics', 'Personal Finance'),
+        m(Personal_Finance_Investing, nonfiction, 'Business & Economics', 'Investments & Securities'),
+        m(Real_Estate, nonfiction, 'Business & Economics', 'Real Estate'),
+        m(Personal_Finance_Business, nonfiction, 'Business & Economics'),
+
+        # Parenting & Family
+        m(Parenting, nonfiction, 'Family & Relationships', 'Parenting'),
+        m(Family_Relationships, nonfiction, 'Family & Relationships'),
+
+        # Food & Health
+        m(Bartending_Cocktails, nonfiction, 'Cooking', 'Beverages'),
+        m(Health_Diet, nonfiction, 'Cooking', 'Health & Healing'),
+        m(Health_Diet, nonfiction, 'Health & Fitness'),
+        m(Vegetarian_Vegan, nonfiction, 'Cooking', 'Vegetarian & Vegan'),
+        m(Cooking, nonfiction, 'Cooking'),
+
+        # History
+        m(African_History, nonfiction, 'History', 'Africa'),
+        m(Ancient_History, nonfiction, 'History', 'Ancient'),
+        m(Asian_History, nonfiction, 'History', 'Asia'),
+        m(Civil_War_History, nonfiction, 'History', 'United States', re.compile('^Civil War')),
+        m(European_History, nonfiction, 'History', 'Europe'),
+        m(Latin_American_History, nonfiction, 'History', 'Latin America'),
+        m(Medieval_History, nonfiction, 'History', 'Medieval'),
+        m(Military_History, nonfiction, 'History', 'Military'),
+        m(Renaissance_Early_Modern_History, nonfiction, 'History', 'Renaissance'),
+        m(Renaissance_Early_Modern_History, nonfiction, 'History', 'Modern', re.compile('^1[678]th Century')),
+        m(Modern_History, nonfiction, 'History', 'Modern'),
+        m(United_States_History, nonfiction, 'History', 'United States'),
+        m(World_History, nonfiction, 'History', 'World'),
+        m(World_History, nonfiction, 'History', 'Civilization'),
+        m(History, nonfiction, 'History'),
+
+        # Hobbies & Home
+        m(Antiques_Collectibles, nonfiction, 'Antiques & Collectibles'),
+        m(Crafts_Hobbies, nonfiction, 'Crafts & Hobbies'),
+        m(Gardening, nonfiction, 'Gardening'),
+        m(Games, nonfiction, 'Games'),
+        m(House_Home, nonfiction, 'House & Home'),
+        m(Pets, nonfiction, 'Pets'),
+
+        # Entertainment
+        m(Film_TV, nonfiction, 'Performing Arts', 'Film & Video'),
+        m(Film_TV, nonfiction, 'Performing Arts', 'Television'),
+        m(Music, nonfiction, 'Music'),
+        m(Performing_Arts, nonfiction, 'Performing Arts'),
+
+        # Reference & Study Aids
+        m(Dictionaries, nonfiction, 'Reference', 'Dictionaries'),
+        m(Foreign_Language_Study, nonfiction, 'Foreign Language Study'),
+        m(Law, nonfiction, 'Law'),
+        m(Study_Aids, nonfiction, 'Study Aids'),
+        m(Reference_Study_Aids, nonfiction, 'Reference'),
+        # m(Reference_Study_Aids, nonfiction, 'Linguistics'),
```

Comment:
I think you can get rid of this line? or were you thinking of mapping linguistics to something else?

Reply:
I was thinking of it, but it turned out to be handled by "Language Arts & Disciplines".